docs/communication.md

```diff
@@ -5,7 +5,7 @@ Here are some of the major pieces:
 
 ## Discussion Forum
 
-We have [a GitHub Discussions Forum](https://discuss.executablebooks.org/) for all of our GitHub repositories (rather than using one forum per repository).
+We have [a GitHub Discussions Forum](https://github.com/executablebooks/meta/discussions) for all of our GitHub repositories (rather than using one forum per repository).
```

Comment:
```suggestion
We have [a GitHub Discussions Forum](https://github.com/orgs/executablebooks/discussions) for all of our GitHub repositories (rather than using one forum per repository).
```